Annotato/Annotato/ViewModels/Document/DocumentViewModel.swift

```diff
@@ -151,14 +151,17 @@ extension DocumentViewModel {
     }
 
     func removeAnnotation(annotation: AnnotationViewModel) {
-        model.removeAnnotation(annotation: annotation.model)
-        annotations.removeAll(where: { $0.id == annotation.model.id })
-
+        removeAnnotationWithoutPersistence(annotation: annotation)
         Task {
             await annotationsPersistenceManager.deleteAnnotation(annotation: annotation.model)
         }
     }
 
+    func removeAnnotationWithoutPersistence(annotation: AnnotationViewModel) {
```

Comment:
```suggestion
    func removeAnnotation(annotation: AnnotationViewModel) {
```

The function does not need to know what it is not doing. You can rename the other function to `deleteAnnotation`? Can't really think of a better name for now

Reply:
Ok changed